threedi_api_qgis_client/widgets/wizard.py

```diff
@@ -0,0 +1,323 @@
+import os
+import sys
+import json
+from math import ceil
+from datetime import datetime
+from qgis.PyQt.QtSvg import QGraphicsSvgItem
+from qgis.PyQt import uic
+from qgis.PyQt.QtGui import QColor
+
+from qgis.PyQt.QtWidgets import QWizardPage, QWizard, QGridLayout, QGraphicsScene, QSizePolicy, QInputDialog
+try:
+    import pyqtgraph as pg
+except ImportError:
+    main_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    deps_path = os.path.join(main_dir, "deps", "pyqtgraph-0.10.0-py3-none-any.whl")
+    sys.path.append(deps_path)
+    import pyqtgraph as pg
```

Comment:
I would put this wrapper to the separate file like ./custom_imports.py, so we can use it from multiple files

Reply:
Done.